test/prototype/moe_training/test_fsdp_tp.sh

```diff
@@ -0,0 +1 @@
+torchrun --nproc_per_node=4 --local-ranks-filter=0 -m pytest test/prototype/moe_training/test_fsdp_tp.py -s
```

Comment:
at this point I'd also add `test_everything.sh` which runs all of the shell scripts, same as we do for float8 training